projects/RabbitMQ.Client/client/api/BinaryTableValue.cs

```diff
@@ -38,7 +38,7 @@
 //  Copyright (c) 2007-2020 VMware, Inc.  All rights reserved.
 //---------------------------------------------------------------------------
 
-namespace RabbitMQ.Client.Impl
+namespace RabbitMQ.Client.client.api
```

Comment:
The namespace this client uses for public API bits is `RabbitMQ.Client`, not `RabbitMQ.Client.client.api`

Reply:
Yes thanks missed that. Now it's fixed and only the file BinaryTablevalue is changed